docs/api/power-monitor.md

```diff
@@ -8,11 +8,11 @@ Process: [Main](../glossary.md#main-process)
 
 The `powerMonitor` module emits the following events:
 
-### Event: 'suspend' _macOS_ _Windows_
+### Event: 'suspend' _Linux_ _macOS_ _Windows_
```

Comment:
We only add platforms when there's partial support - this can now just be `### Event: 'suspend'`